data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,368 @@
+			`	"I'm Lucy," says the human woman.`
+			`	"And I'm Meroot," adds the Hai woman. "I was on Newhome walking to my research lab when some men pulled me into an alley and zapped me with a tazer. I woke up in the brig of a space ship."`
+			`	"My experience was similar. I was walking from my biolab to my house on Serpens. Some men jumped out from behind a dune and zapped me."`
+			`	Meroot continues, "They took us both here. I was able to struggle out of my handcuffs and sneak the keys from the guard. We escaped together, and we've been hiding for weeks."`
+			`	"Please take us to <destination>"`
+			choice
+				`	"Sure. Let's go."`
+				`	"No, you'll have to find another pilot."`
+					decline
+			`	The two women put on clothing that thoroughly covers their person. They lead you through a series of ducts and alleys back to the building where your ship is parked. When you have fifty meters left to go, someone shoots Meroot with a stun gun. You and Lucy manage to carry her to your ship and lift off. It appears you're being followed into orbit.`
+				launch
+	on complete
+		log `Rescued a pair of researchers: a Hai woman named Meroot and a human woman named Lucy. They were kidnapped by men who brought them to Canyon. Returned them to <destination> where they now live together. They don't know the identities of the kidnappers.`
+		payment 315000
+		"hai slave prereq" ++
+		dialog `Meroot and Lucy thank you for your help and give you <payment>. Lucy decides to live in Hai space with Meroot, "It's safer here." The two don't know who the men were that kidnapped them.`
+	npc
+		government "Bounty Hunter that Won't Enter Hai Space"
+		personality plunders disables unconstrained
+		fleet "Large Core Pirates" 1
+		fleet "Small Core Pirates" 1
+			
+
+mission "Hai Rescue: Gipeep"
+	landing
+	name "Return Gipeep"
+	description "Transport the Hai you rescued to <destination>. Payment is <payment>."
+	source "Maelstrom"
+	destination "Greenbloom"
+	passengers 1
+	to offer
+		"combat rating" > 50
+		not "event: hai-human treaty signed"
+		has "Unwanted Cargo: done"
+		has "Hai Leaks Response 6: active"
+		# Some subset of these are restricted so you can't pick up enough of them to finish the quest at the same time as starting it.
+	on offer
+		conversation
+			`You walk by some cargo and hear loud thumping coming from a locked crate. You notice a key nearby. Do you want to check it out?`
+			choice
+				`	(Examine the crate.)`
+				`	(Leave.)`
+					decline
+			`	The key has a symbol you don't recognize, but it looks like an angry cat in a circle. As you get closer, the ordinary-looking crate starts seeming a little strange. It appears to be well-insulated against sound and temperature, but has concealed breathing holes. You can still hear loud thumping.`
```

Comment:
```suggestion
			`	The key has a symbol you don't recognize, but it looks like an angry cat in a circle. As you get closer, the ordinary-looking crate starts to look a little strange. It appears to be well-insulated against sound and temperature, but has concealed breathing holes. You can still hear loud thumping.`
```
Loud thumping, but it's noticeably sound-containing? That's a little contradictory.

Reply:
Another one that I might attempt a broader re-write.